app/assets/stylesheets/application.css

```diff
@@ -1,17 +1,20 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- *= require foundation_and_overrides
+body{
+  background-color: #FEFAE0;
+}
 
- */
+h2{
+  margin: 20px;
+  font-family: 'Fredoka One', serif;
+}
+
+.form-container{
+  border: solid;
+  border-radius: 1em;
+  margin: 20px;
+  padding: 10px 20px;
```

Comment:
nothing that needs to change, but best practice is to be consistent in the type of sizing that you choose, either always using px or em rather than a mix (em is generally preferred as it's more relative than a static measurement)